boringtun/src/ffi/mod.rs

```diff
@@ -322,16 +319,3 @@ pub unsafe extern "C" fn wireguard_stats(tunnel: *mut Tunn) -> stats {
         reserved: [0u8; 56],
     }
 }
-
-/// Performs an internal benchmark, and returns its result as a C-string.
-#[no_mangle]
-pub extern "C" fn benchmark(name: i32, idx: u32) -> *const c_char {
```

Comment:
Excellent. Let's make sure it's also removed from `wireguard_ffi.h`.

Reply:
We need to, after this, add CI checks for wireguard_ffi.h